Formula/pastebinit.rb

```diff
@@ -14,7 +15,7 @@ class Pastebinit < Formula
   end
 
   depends_on "docbook2x" => :build
-  depends_on "python3"
+  depends_on "python"
```

Comment:
The `inreplace` below needs fixing to match this change.

Reply:
It'll work because of the `alias`, as far as I can see, but seems sensible to make the change to avoid confusion?